src/Features/CSharp/Portable/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -114,14 +115,23 @@ private static async Task HandleSingleTypeAsync(CompletionContext context, Seman
                 type = typeArg;
             }
 
+            var showTypeOnlyIfMembers = false;
```

Comment:
this name wasn't really clear to me.

Reply:
When I woke up this morning I had a new thought to remove this.